tools/parse-release-notes

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { GITKEEP_FILE, UNRELEASED_DIR, RELEASE_NOTES_DIR } = require('./constants')
+const { program } = require('commander')
+const yaml = require('js-yaml')
+
+const DEFAULT_LANGUAGE = 'en'
+
+const DEFAULT_NOTE_EN = 'We’ve been working hard behind the scenes to make sure everything is working as it should. If you notice anything that does not work, let us know!\n'
+const DEFAULT_NOTE_DE = 'Wir haben hinter den Kulissen hart gearbeitet, um sicherzustellen, dass alles so funktioniert, wie es soll. Wenn Sie bemerken, dass etwas nicht funktioniert, lassen Sie es uns wissen!\n'
+
+const PLATFORM_ANDROID = 'android'
+const PLATFORM_IOS = 'ios'
+const PLATFORM_WEB = 'web'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+  .option('--ios', 'include release notes for ios')
+  .option('--android', 'include release notes for android')
+  .option('--web', 'include release notes for web. may not be used with --android or --ios flags')
+  .option('--production', 'whether to hide extra information, e.g. issue keys, hidden notes and platforms and prepare the notes for a store. may not be used with multiple platforms.')
+  .option('--destination <destination>', 'if specified the parsed notes are saved to the directory')
+  .requiredOption('--source <source>', 'the directory of the release notes to parse', `${RELEASE_NOTES_DIR}/${UNRELEASED_DIR}`)
+  .requiredOption('--language <language>', 'the language of the release notes to parse', DEFAULT_LANGUAGE)
+
+const formatNotes = ({ notes, language, production, platformName }) => {
+  if (notes.length === 0) {
+    return ''
+  }
+
+  const formattedNotes = notes
+    .map(note => {
+      const localizedNote = note[language] || note[DEFAULT_LANGUAGE]
+      return production ? `* ${localizedNote}` : `* ${note.issue_key}: ${localizedNote}`
+    })
+    .join('\n')
+
+  return platformName ? `\n${platformName}:\n${formattedNotes}` : `${formattedNotes}`
```

Comment:
```suggestion
  return platformName ? `${platformName}:\n${formattedNotes}` : `${formattedNotes}`
```

Reply:
Actually, this is fine :) Otherwise we have `Release notes:ios:` instead of
`Release notes:
ios:`